src/ui/shared/CardHeader.tsx

```diff
@@ -330,6 +333,7 @@ const CardHeader: React.FC<Props> = ({
 						<Flex alignItems="center" spaceSize="m">
 							{showShareButton && (
 								<Button
+									ariaLabel={shareButtonLabel ?? t('Share')}
```

Comment:
Should we not only set `ariaLabel` if `shareButtonLabel` is undefined? So something like `shareButtonLabel ? undefined : t('Share')`?